recipes/pyorc/meta.yaml

```
@@ -0,0 +1,60 @@
+package:
+  name: pyorc
+  version: "0.4.0"
+
+source:
+  url: https://pypi.io/packages/source/p/pyorc/pyorc-0.4.0.tar.gz
+  sha256: af56c50dd0545b451a37d38e73e02405f332e91779cf9ab1c8ae3e61b9eb0c90
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: |
+    {{ PYTHON }} setup.py build_orc --download-only=true
+    {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36 or win]
```

We don't build for `py<36`, thus can be omitted here.

```suggestion
  skip: True  # [win]
```